app/src/main/java/com/oxeanbits/forecastchart/model/LineDataCreator.kt

```diff
@@ -0,0 +1,10 @@
+package com.oxeanbits.forecastchart.model
+
+import com.github.mikephil.charting.data.Entry
+
+data class LineDataCreator (
+    val values: ArrayList<Entry>,
```

Comment:
`Entry` is a pair, right? What type we can pass? String, Decimal, Integer?

Reply:
`Entry` is a pair of Floats (X and Y axis).